docs/getting-started/v3-migration.md

```diff
@@ -86,10 +86,16 @@ Animation system was completely rewritten in Chart.js v3. Each property can now
 
 ### Removed
 
+* `Chart.borderWidth`
 * `Chart.chart.chart`
 * `Chart.Controller`
+* `Chart.innerRadius`
+* `Chart.offsetX`
+* `Chart.offsetY`
+* `Chart.outerRadius`
```

Comment:
Looks like `outerRadius` and `innerRadius` are still set

Reply:
Which would mean they also don't need to be removed from the test

Reply:
That was my bad, misread the code and still saw them getting set. It's just to only the scale now.